config/config.go

```diff
@@ -11,6 +11,8 @@ import (
 
 const ExchangeURLEnvvarName = "HZN_EXCHANGE_URL"
 const FileSyncServiceCSSURLEnvvarName = "HZN_FSS_CSSURL"
+const VaultURLEnvvarName = "VAULT_ADDR"
```

Comment:
openhorizon env vars are always prefixed with HZN_

Reply:
@dabooz the variable served one more purpose. To use the vault-cli, this variable needs to be set in the agreement bot.